key-chord.el

```diff
@@ -205,6 +205,9 @@
   "Max time delay between two press of the same key to be considered a key chord.
 This should normally be a little longer than `key-chord-two-keys-delay'.")
 
+(defvar key-chord-safety-interval 0.08
```

Comment:
Later you replace that with two variables. Why not do that right away?

Reply:
OK, I fixup commit.